<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkupDlg</class>
 <widget class="QDialog" name="MarkupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="8">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Markup attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="plainTags">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Tags</string>
        </property>
        <property name="placeholderText">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushAddTag">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>tags</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkGenID">
          <property name="text">
           <string>Auto UID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkFiles">
          <property name="text">
           <string>Add to FILES</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTags">
          <property name="text">
           <string>Add to TAGS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkOutline">
          <property name="text">
           <string>Add to OUTLINE</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelId">
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>ID/Sig</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushClear">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clr</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineId">
          <property name="toolTip">
           <string>Path/ID</string>
          </property>
          <property name="placeholderText">
           <string>Path/ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushGen">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Generate unique ID</string>
          </property>
          <property name="text">
           <string>Gen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Additional attrinutes for tree nodes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="textText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Description</string>
        </property>
        <property name="placeholderText">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineName">
        <property name="toolTip">
         <string>Title (alias for tree node)</string>
        </property>
        <property name="placeholderText">
         <string>Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="ColorButton" name="clrMark">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select color...</string>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="checkInline">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Insert #color into source code</string>
     </property>
     <property name="text">
      <string>Inline</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboPredefColors">
     <property name="toolTip">
      <string>Select predefined color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboRecentColors">
     <property name="toolTip">
      <string>Select recent color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>../src/gui/widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
